lib/ts/recipe/authRecipe/components/feature/authPage/authPage.tsx

```
@@ -172,7 +172,8 @@ const AuthPageInner: React.FC<AuthPageProps> = (props) => {
             .getCurrentDynamicLoginMethods({ userContext })
             .then(
                 (loginMethods) => setLoadedDynamicLoginMethods(loginMethods),
-                (err) => rethrowInRender(err)
+                // eslint-disable-next-line @typescript-eslint/no-empty-function
+                () => {}
```

Please add a comment explaining why we are ignoring the exception here.

---

Added!